.changeset/empty-poems-hug.md

```diff
@@ -0,0 +1,9 @@
+---
+"@osdk/views-client-react.unstable": minor
+"@osdk/views-client.unstable": minor
+"@osdk/views-api.unstable": minor
+"@osdk/client": patch
+"@osdk/api": patch
```

Comment:
Should these last two be here?

Reply:
this feels like it came from something on `main`..?

Reply:
hmm, looks like maybe rebase got you? I bet they cut a beta and you still have this in your tree